src/wallet/wallet.ts

```diff
@@ -1709,6 +1722,33 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     const tx = await this.prepareCreateNewToken(name, symbol, amount, newOptions);
     return this.handleSendPreparedTransaction(tx);
   }
+
+  /**
+   * Check if websocket connection is enabled
+   * 
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   * 
+   * @returns {boolean} If wallet has websocket connection enabled
+   */
+  isWsEnabled(): boolean {
+    return this._isWsEnabled;
+  }
+
+  /**
+   * This is a test only method, it should not be used in production
+   */
+  async callOnWalletReadyForTest() {
```

Comment:
Can't we use jest to call `onWalletReady` instead of adding a test method in the facade?

Reply:
It is not trivial, but done.